docs/overview/script-storage.md

```diff
@@ -25,13 +25,12 @@ Office Scripts that are saved to a SharePoint site are owned by your team. You a
 
 To load a script from SharePoint, go to **All scripts** and select **View more scripts** at the bottom of the list. This brings up a file picker where you can choose **.osts** files from any SharePoint site to which you have access. Note that scripts from SharePoint that you've already opened will be displayed in the list of recent scripts.
 
-To save a script to SharePoint, go to the **More options (…)** menu and select **Save as**. This opens a file picker where you can select folders in your SharePoint site. Saving to a new location creates a copy of the script at that location. The original version is still on your OneDrive or other SharePoint location.
+To move a script to SharePoint, open a script in the editor and click on the script name. This will open a callout. In the callout, click the move button. This opens a file picker where you can select folders in your SharePoint site. Saving to a new location moves the script to that location. This can take up to a minute.
```

Comment:
Similarly, this suggestion is to try and make the UI choices more relatable to the user.

```suggestion
To move a script to SharePoint, open a script in the **Code Editor** and click on the script name, as if you're renaming it. In the callout, click the **Move** button. This opens a file picker. Select the destination folder in your SharePoint site. Moving the script to the new location can take up to a minute.
```